hashintel/hash
# H-1750, H-1809: Allow filtering drafts by containing `Web`

## 🌟 What is the purpose of this PR?




This PR adds a "Web" filter to the drafts page, allowing to filter draft entities by the web they were created in.

Driveby changes:
- Change icon for `Drafts` in left-hand sidebar to `quill` to match Figma (H-1809)

## 🔗 Related links





- H-1750
- H-1809

## 🔍 What does this change?






## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## ⚠️ Known issues



## 🐾 Next steps



## 🛡 What tests cover this?



Manual testing

## ❓ How to test this?



1. Create a draft entity in an org's web that the user has access to (I do this by modifying the `createEntity` function in `apps/hash-api/src/graph/knowledge/primitive/entity.ts` to create drafts by default, and directly using the entity editor to create draft entities)
2. Verify that both draft entities in the user's and in the org's web appear on the `/drafts` page
3. User the "Web" filter to filter the entities based on the web they were created in

## 📹 Demo



https://github.com/hashintel/hash/assets/42802102/56f4ada5-e2f8-4f81-8dcc-b7d5c42200d6
